index.js

```diff
@@ -19,9 +19,9 @@ server.use(express.static(path.join(__dirname, 'build')));
 server.use('/api', require('./routes'));
 
 // by default serve up the react app if we don't recognize the route
-//server.use((req, res, next) => {
-//  res.sendFile(path.join(__dirname, 'build', 'index.html'))
-//});
+server.use((req, res, next) => {
+  res.sendFile(path.join(__dirname, 'build', 'index.html'))
```

Comment:
not here, but please set the title on your index.html page to be something significant! Not just "Full-Stack App"